add-on/panels/js/sidebar.jsx

```diff
@@ -148,22 +165,21 @@ loop.sidebar = (function(mozL10n) {
       });
 
       ReactDOM.render(<SidebarControllerView
-                        activeRoomStore={activeRoomStore}
                         dispatcher={dispatcher} />, document.querySelector("#main"));
 
-      var locationData = sharedUtils.locationData();
-      var hash = locationData.hash.match(/#(.*)/);
-
       dispatcher.dispatch(new sharedActions.SetupWindowData({
-        windowId: "id-test",
-        roomToken: hash[1],
+        roomToken: roomToken,
         type: "room"
```

Comment:
I think we can drop it. It was when we had the call vs room mechanisms, and we're unlikely to need that soon.